### PR TITLE
test(e2e): add skeleton for pipeline e2e test

### PR DESCRIPTION
**Pull request by efekarakus**
This initial e2e/pipeline pkg just creates a codecommit repository
where we can push the service code. Once this PR is merged and we know
that it works with CodeBuild, we'll send a follow-up PR to actually run
the "copilot pipeline" commands.

- [x] Tested the changes against my AWS account and local machine

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by huanjani — APPROVED** [review]
Thank you for doing this!

File: e2e/pipeline/pipeline_test.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	Context("setup CodeCommit repository", func() {
+		var cloneURL string
+		It("creates the codecommit repository", func() {
+			url, err := aws.CreateCodeCommitRepo(repoName)
+			Expect(err).NotTo(HaveOccurred())
+			cloneURL = url
+		})
+
+		It("clones the repository", func() {
+			endpoint := strings.Split(cloneURL, "https://")[1]
```
Comment (iamhopaul123) [inline e2e/pipeline/pipeline_test.go]: Why not use `strings.TrimPrefix`?

**Review by iamhopaul123 — APPROVED** [review]
LGTM just a nit

File: e2e/pipeline/pipeline_test.go (hunk 1 above)
Comment (efekarakus) [reply e2e/pipeline/pipeline_test.go]: I forgot about that! I'll update it 👍

**Pull request merged**